src/deserialize.ts

```diff
@@ -35,3 +37,14 @@ export const deserializeDkgRound2 = (data?: Buffer) => {
     publicPackage,
   }
 }
+
+export const deserializeReviewTx = (data?: Buffer) => {
```

Comment:
This is not a buffer, but a ResponsePayload. It implements the ByteStream class. Check how we read in other cases!

Reply:
it is defined as a buffer in Round2?
`export const deserializeDkgRound2 = (data?: Buffer)`

Reply:
It is correct! My bad!